packages/abstractions/test/common/requestInformation.ts

```diff
@@ -190,6 +192,15 @@ describe("RequestInformation", () => {
 		assert.equal(methodCalledCount, 1);
 	});
 
+	it("should correctly handle Guid type query parameter", () => {
```

Comment:
Would it be possible to add tests for instances for `TimeOnly` and `DateOnly` types that could be in the URL as well? Similar to https://github.com/microsoft/kiota-abstractions-python/pull/278/files

Reply:
This is fixed now @andrueastman